roles/network_plugin/cilium/defaults/main.yml

```diff
@@ -39,3 +39,12 @@ cilium_enable_legacy_services: false
 # Deploy cilium even if kube_network_plugin is not cilium.
 # This enables to deploy cilium alongside another CNI to replace kube-proxy.
 cilium_deploy_additionally: false
+
+# Auto direct nodes routes can be used to advertise pods routes in your cluster
+# without any tunelling (with `cilium_tunnel_mode` sets to `disabled`).
+# This works only if you have a L2 connectivity between all your nodes.
+# You wil also have to specify the variable `cilium_native_routing_cidr` to
+# make this work. Please refer to the cilium documentation for more
+# information about this kind of setups.
+cilium_auto_direct_node_routes: false
+cilium_native_routing_cidr: ""
```

Comment:
According to https://docs.cilium.io/en/v1.8/concepts/networking/routing/?highlight=auto-direct-node-routes#configuration this addition seems good for me.